polkadot.json

```
@@ -7,68 +7,68 @@
         "id": 1,
         "imageUrl": "https://i.imgur.com/vF7XfK8.png",
         "title": "Meet the next-generation blockchain",
-        "details": "Welcome to the world of Polkadot, a blockchain platform that enables communication between different blockchains - created by Gavin Wood, one of the co-founders of Ethereum. It’s able to connect with different blockchains to form an interoperable, decentralized web."
+        "details": "Welcome to the world of Polkadot, a platform for blockchains and Web3 applications - created by Gavin Wood, one of the co-founders of Ethereum. It is a vision towards a free, open, and decentralized world computer."
       },
       {
         "id": 2,
         "imageUrl": "https://i.imgur.com/9BQhqPY.png",
         "title": "Building blocks of Polkadot",
-        "details": "Think of Polkadot like building with Legos - you can pick and choose the pieces you want to create something unique! Imagine building a hyper-optimized blockchain, called a parachain, with specific characteristics to fit your exact need! All this is possible with Polkadot!"
+        "details": "Polkadot is designed to solve 3 fundamental problems in the blockchain space: scalability, security, and interoperability. The Polkadot ecosystem is composed of many independent web3 applications, like blockchains and DApps. Imagine building or using hyper-optimized blockchains, called a parachains, designed to fit to your exact needs! All of this is possible with Polkadot!"
       },
       {
         "id": 3,
-        "imageUrl": "https://i.imgur.com/XbI361a.png",
-        "title": "Unleash Versatility",
-        "details": "Plus, because of how it's designed, Polkadot can do things other blockchains can't, like handle lots of transactions at once and connect with external blockchain systems too."
+        "imageUrl": "https://i.imgur.com/kdwrLg5.png",
+        "title": "Scalability solved",
+        "details": "Polkadot scales Web3 applications through parallelization and sharding. Transactions to different applications can be executed in parallel, and data from these applications are also hosted in their own blockchain. This allows applications to grow quickly without affecting the fees or speeds of other applications."
```

As usual I am prone to adding more and more details 😁


```suggestion
        "details": "Polkadot scales Web3 applications through parallelization and sharding. Transactions to different applications can be executed in parallel, and data from these applications are also hosted in their own blockchain. This allows applications to grow quickly without affecting the fees or speeds of other applications. All of this is achieved under the umbrella of the economic security of the Polkadot relay chain, delivering scalability and security together 🤝"
        
Or else I would argue the same phrase would apply 100% to Cosmos chains as well. 
```